src/mapdata/roomfilter.cpp

```diff
@@ -0,0 +1,54 @@
+#include "roomfilter.h"
+#include <assert.h>
+#include <errno.h>
+
+const char * RoomFilter::parse_help = "Parse error; format is: [-(name|desc|note|exits|all)] text\r\n";
```

Comment:
This should probably also be documented somehow. Or maybe just mention _dirs  -help ?